src/utils/makeReactNativeConfig.js

```diff
@@ -211,16 +228,27 @@ const getDefaultConfig = ({
   };
 };
 
-/**
- * Creates an array of configs based on changing `env` for every
- * platform and returns
- */
-function makeReactNativeConfig(
-  userWebpackConfig: WebpackConfigFactory,
+/** 
+ * Return React Native config
+ * 
+ * @deprecated
+*/
+function DEPRECATEDMakeReactNativeConfig(
+  userWebpackConfig: DEPRECATEDWebpackConfigFactory,
   options: ConfigOptions,
-  platform: 'ios' | 'android'
+  platform: string
```

Comment:
https://github.com/callstack/haul/blob/8f84cbb077b2a17b8ae9a921ef32b4d590444810/src/types.js#L56

Reply:
Yeah, I had some problem with type inheritance. I will try once more. 

Flow claimed that he can't allow a string as an enum.